liquibase-extension-testing/src/main/java/liquibase/extension/testing/testsystem/core/MSSQLTestSystem.java

```diff
@@ -20,8 +20,10 @@ public MSSQLTestSystem(Definition definition) {
 
     @Override
     protected @NotNull DatabaseWrapper createContainerWrapper() {
-        return new DockerDatabaseWrapper(new MSSQLServerContainer(
-                DockerImageName.parse(getImageName()).withTag(getVersion())),
+        final MSSQLServerContainer container = new MSSQLServerContainer(DockerImageName.parse(getImageName()).withTag(getVersion()));
+        container.withUrlParam("encrypt", "false");
```

Comment:
@suryaaki2 @StevenMassaro @XDelphiGrl here is the hack @nvoxland had to make this work for now